packages/beacon-node/src/metrics/metrics/beacon.ts

```diff
@@ -335,6 +335,34 @@ export function createBeaconMetrics(register: RegistryMetricCreator) {
       }),
     },
 
+    peerDas: {
```

Comment:
please see [here](https://github.com/ChainSafe/lodestar/blob/8aa1ad8d30da315d04efaa3dbfad9bdc23e5ccff/CONTRIBUTING.md?plain=1#L190-L194) for naming conventions, I will leave some suggestions below

Reply:
Looks like the main convention for variables is to not suffix them, so updated these to use `Time` and remove `Seconds`. I think the metric names are okay, but probably would have to get those changed upstream if we wanted to

Reply:
> I think the metric names are okay, but probably would have to get those changed upstream if we wanted to

metric names are different, it's as per prometheus convention to suffix with `_seconds` to make it easier for consumers to know the unit, internally, we only use seconds, so the suffix is meaningless